handlers/config_handler.py

```diff
@@ -37,8 +37,11 @@ def title_image(self):
     def title(self):
         return self.root["title"]
 
+    def secret_key(self):
+    	return self.root["secret_key"]
+
     def to_json(self):
```

Comment:
Rename this to include that it will return setups. `to_json` could make someone think it returns the full configuration. It doesn't

Reply:
Renamed the function to setup()